hbase/src/main/scala/akka/stream/alpakka/hbase/HTableSettings.scala

```diff
@@ -76,8 +76,5 @@ object HTableSettings {
                 tableName: TableName,
                 columnFamilies: java.util.List[String],
                 converter: java.util.function.Function[T, java.util.List[Mutation]]): HTableSettings[T] =
-    HTableSettings(conf,
-                   tableName,
-                   columnFamilies.asScala.to[immutable.Seq],
-                   converter.asScala(_).asScala.to[immutable.Seq])
+    HTableSettings(conf, tableName, columnFamilies.asScala.toIndexedSeq, converter.asScala(_).asScala.toIndexedSeq)
```

Comment:
Might be nice to have the newlines here?

Reply:
Its scalafmt. :)